### PR TITLE
Test for files with same filename with different path in trashbin

### PR DESCRIPTION
**Pull request by dpakach**
## Description
Added acceptance test for checking if multiple files with same filename and different path are listed in trashbin  after deleting.

## Related Issue

- Related to #31038 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/lib/FilesPageElement/TrashBinFileRow.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * ownCloud
+ *
+ * @author Artur Neumann <artur@jankaritech.com>
+ * @copyright Copyright (c) 2017 Artur Neumann artur@jankaritech.com
```
Comment (phil-davis) [inline tests/acceptance/features/lib/FilesPageElement/TrashBinFileRow.php]: Make yourself the author and change the year.

**Review by phil-davis — COMMENTED** [review]
This looks good. It re-uses code in FilesPageBasic nicely.
There are just 2 comments to look at.

**Comment by dpakach** [comment]
@phil-davis  I made those changes. Please take a look.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport ``stable10`` #32836

**Pull request merged**